platform/admin/core/admin-core-logviewer/README.md

```diff
@@ -0,0 +1,58 @@
+<!--
```

Comment:
We didn't use any scaffolding for this project and I don't believe that we are using it anywhere else in ddf's js projects.